project-resources/github-continuity-policy.md

```diff
@@ -0,0 +1,31 @@
+# OpenJS Github Continuity Policy
+
+## Summary
+
+This policy is intended to ensure that OpenJS GithHub projects remain accessible and manageable. 
+
+It addresses the following use cases:
+
+- Archiving projects when maintainers are no longer reachable
+- Managing projects and users when maintainers are inactive or unreachable
+- Adding or removing admins in emergency situations
+
+## Continuity Options for Projects
+
+### Option 1: Add The Executive Director As An Admin
+
+If you select this option, you will add the OpenJS Foundation executive director (currently https://github.com/rginn) 
```

Comment:
What happens in this case if the executive director changes?

Reply:
the github permissions and this document would need to be updated